src/targets/gpu/pack_int8_args.cpp

```diff
@@ -156,6 +156,12 @@ void pack_int8_args::apply(module& m) const
         }
         else if(ins->name() == "gpu::quant_convolution")
         {
+            auto val = ins->get_operator().to_value();
+            if(not val.at("int8_x4_format").to<bool>())
+            {
+                return;
```

Comment:
This should do `continue`(actually the other place where there is a `return` is a bug).

Reply:
Ah yes, I think you mentioned about this when we walked through the code and I just forget about it in the implementation. I'll amend to `continue` in both places.

Reply:
Addressed in d5eb524